### DISCUSSION TITLE
Docker not working

### DISCUSSION THREAD

**Discussion by tonysamperi** [community/community#26789] [Actions]
<p>Today my GitHub actions isn’t able to complete succesfully because suddenly there are issues starting the docker container.<br>
The strange thing is that I didn’t change anything except the source code.<br>
I really can’t figure out why I’m getting an error. Could it be a cache problem?</p>
<p>This is the command:</p>
<pre><code class="lang-auto">docker run --rm -v ${pwd}:/ts-luxon -w /***/ts-luxon ***/ts-luxon ./docker/workflow.sh
</code></pre>
<p>But it fails with this error:</p>
<pre><code class="lang-auto">docker: Error response from daemon: OCI runtime create failed: invalid mount {Destination::/ts-luxon Type:bind Source:/var/lib/docker/volumes/edd728cda79ecb1813555a3edf503bac68dd0c7a7d5ca9b34c05578641a64665/_data Options:[rbind]}: mount destination :/ts-luxon not absolute: unknown.
</code></pre>
<p>I discovered that there was an incident related, but it seems to be solved now.<br>
I think there might still be something wrong.</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="tonysamperi" data-post="1" data-topic="181304">
<div class="title">
<div class="quote-controls"></div>
 tonysamperi:</div>
<blockquote>
<p><code>docker run --rm -v ${pwd}:/ts-luxon -w /***/ts-luxon ***/ts-luxon ./docker/workflow.sh</code></p>
</blockquote>
</aside>
<p>If you want to mount the current working directory, you need to either use <code>${PWD}</code> (capitals) for the environment variable, or <code>$(pwd)</code> to run the <code>pwd</code> command and use its output. :wink:</p>

**Comment by tonysamperi** [comment]
<p>Thank you! I’ll try right away. But why did it use to work??</p>

**Comment by tonysamperi** [comment]
<p>I’m speechless. It works. Thanks a lot! It would have taken years for me to find that!</p>

**Comment by arrudacaio** [comment]
<p>In my case, I’m using git bash and I needed to change <strong>$PWD</strong> to <strong>/$(PWD)</strong></p>
